app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -47,8 +47,10 @@ class WallpaperManager(private val settings: Settings) {
                 AppCompatDelegate.MODE_NIGHT_NO
             }
         } else {
-            updateThemePreference(followDeviceTheme = true)
-            AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM
+            // For the default wallpaper, there is not need to adjust the theme,
+            // as we want to allow users decide which theme they want to have.
+            // The default wallpaper adapts to whichever theme the user has.
+            return
```

Comment:
I think this means you could remove the `followDeviceTheme` parameter to `updateThemePreference`

Reply:
Yeah, that's true, thanks for catching this!